src/linter/messages.ts

```diff
@@ -175,6 +178,25 @@ export const MESSAGE_INFO = {
 		details: ({details}: {details: string}) => details,
 	},
 
+	[MESSAGE.DEPRECATED_VIEW_CONFIG]: {
+		severity: LintMessageSeverity.Error,
+		ruleId: RULES["no-deprecated-api"],
```

Comment:
I know we're trying to consolidate those, but I'm still wondering whether we need to distinguish between API deprecation, partial deprecation and whole files/patterns deprecation...

Just food for thought and something we might want to discuss.
My point is that in the future we might want to do some filtering based on ruleId and `no-deprecated-api` would have a significant percentage of the findings

Reply:
Yes, absolutely. Similarly in https://github.com/SAP/ui5-linter/pull/316

I find it hard to come up with rule names that generalize some of our messages without getting too specific again. Ultimately most of these findings are reported because something got "deprecated", but it's not really an API. So "no-deprecated-view-type"? That's already almost too specific and can only cover few of the messages.

We should look at the whole list we have now and re-group things ideally before the 1.0 release.

Reply:
FW, FV and me discussed this briefly. Looking at it from an app developer perspective it seems reasonable to collect all deprecations under a single (or very few) rules. We couldn't identify any benefit from differentiating between "deprecated views", "deprecated properties", "deprecated parameters" etc.

Ultimately a developer will have to deal with all of these cases very similarly.